readmore.md

```diff
@@ -0,0 +1,186 @@
+# MuSig2 Implementation with libsecp256k1
+
+This is a MuSig2 implementation using the libsecp256k1 library for EC operations.
+
+To install the library, follow the directions in [libsecp256k1](https://github.com/bitcoin-core/secp256k1).
+The library provides an optimized C library for ECDSA signatures and secret and public key operations on curve ecp256k1, as well as usage examples including ECDSA and Schnorr signatures.
+
+This implementation requires configuring the libsecp256k1 with an additional flag `--enable-module-schnorrsig` as stated in [libsecp256k1](https://github.com/bitcoin-core/secp256k1).
+
+Run the example with examplemusig2.c
+
+```
+make ctest
+./ctest_run/ctest
```

Comment:
When looking at this, I realise that if we name the file as example, the make command (and file we create) should also refer to it as example.